CONTRIBUTING.md

```diff
@@ -23,3 +23,28 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
 If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) file for steps that will get you up and running.
 
 Thanks for helping us make RSpec better!
+
+## `Small` issues
+
+Small issues are the ones that we be believe are best suited for new
```

Comment:
Personally I feel these lines would read better as `These issues ...` as its a bit repetitious at the moment.